src/clusterfuzz/stacktraces/__init__.py

```diff
@@ -377,6 +377,14 @@ def remove_lkl_kernel_times_and_set_params(self, state, crash_data):
           state.lkl_kernel_build_id = match.group(2)
     return result
 
+  @staticmethod
+  def split_stacktrace(stacktrace: str):
+    """Split stacktrace by line, and handle special cases with regex."""
+    stacktrace = re.sub(CONCATENATED_SAN_DEADLYSIGNAL_REGEX,
```

Comment:
Add a comment here explaining what this is doing. i.e. handling malformed data where there is a missing newline between the actual crash info and this line.